README.md

```diff
@@ -24,6 +24,8 @@ Instead, I propose that developers cryptographically sign the resources they int
 
 The prototype consists of two signed JavaScript files. One has been modified after it was signed. Only the signed file should execute in the browser. The prototype uses [OpenPGPJs](http://openpgpjs.org), and requires a modern browser. Just view tests/index.html and view the browser console.
 
+**Please note:** In practice, embedding the public key anywhere on the site where the content is also hosted (like in this example code) would be effectively useless. To help mitigate risk, it should instead be kept in a centralized and publicly accessible key store, or preferably, using a public key already retained or cached on the client's computer. That centralized mechanism isn't described here. This is only a *shim* which can be implemented in-browser in order to prevent further tampering by a potential hack on the server-side or MITM attack for plain text connections.
+
```

Comment:
I don't necessarily agree. The signature can coexist with the code, which is fine, because it's there as a sort of integrity check (similar to the original SRI that this is conceptualized from). Basically what @scottsb said here: https://twitter.com/thescottsb/status/676519476154793984 - The idea here is if the file contents were compromised (hence the need to cryptographically verify contents with pub/priv key) then so could the public key as well, if they live in the same place. So the idea here is to distribute the  potential for compromise for which this is originally intended. Also public key servers are already a common/old concept anyway, especially for PGP/GPG.